### ISSUE TITLE
`@stylistic/function-parentheses-space-inside` fix takes the closing parenthesis into an inline comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#114]
Found while fixing #100, in the same sweep as the sibling defect in `function-parentheses-newline-inside`. The same shape as #63, #87, #88 and #89: an inline comment ends only with a line break, so whatever the fix pulls onto its line is commented out. Here it is the closing parenthesis of the function, and both `never` and `always` do it.

### What minimal example or steps are needed to reproduce the bug?

```less
.a {
  transform: translate(
    1px, 2px // c
  );
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-less",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-space-inside": "never"
  }
}
```

### What did you expect to happen?

The whitespace in front of the closing parenthesis holds the line break that closes the comment, so neither option can be satisfied by touching it: `never` would have to delete that line break, and `always` would have to replace it with a single space. The fix should leave the declaration alone and let the warning stand.

### What actually happened?

```less
.a {
  transform: translate(1px, 2px // c);
}
```

With `always` the parenthesis lands inside the comment just the same, one space further along:

```less
.a {
  transform: translate( 1px, 2px // c );
}
```

Either way the closing parenthesis and the semicolon are commented out, the declaration ends nowhere, and no warning survives the run — `--fix` reports a clean pass on a file it has just broken. `postcss-less` reads the output back without complaining, since it keeps a `//` in a value as ordinary text, so nothing downstream catches it either. Less itself does not: for it the line ends at the double slash.

The same fixture under `postcss-scss` is spared for the time being, and only by accident: that syntax keeps a second copy of a value carrying an inline comment and prints it, while the fix is written into the copy it does not print, so the edit is dropped along the way. Whoever repairs that dropping hands this rule the same corruption under SCSS.

The cause is at [`lib/rules/function-parentheses-space-inside/index.js:104-118`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-space-inside/index.js#L104-L118): `valueNode.after` is overwritten with an empty string or with a single space, whatever it held.

### Do you have a proposal to fix the bug?

The shape the cluster settled on: where the text the fix would overwrite ends with an inline comment, report the problem and write nothing. `endsWithInlineComment` in `lib/utils/` already answers that question, and `valueNode.after` — together with whatever comment nodes stand at the end of the function — is what has to be put to it.

`always-single-line` and `never-single-line` are safe by construction: a value holding an inline comment spans more than one line, so neither option applies to it.
